### PR TITLE
LEAN-3961

### PR DESCRIPTION
**Pull request by mbetamony**
1- added some new sections that were used by the import but not available in the api. (abstract-teaser  - floating-element - endnotes) 
2- refactored the categories to support the new type in https://github.com/Atypon-OpenSource/manuscripts-transform/pull/158
3- removed the config route we had for section categories, section categories are now served through templates. To avoid repeating the section category array inside template files, I'm mapping the section-categories and their values when we initialize the templates

### CODE REVIEW

**Comment by mbetamony** [comment]
I made some changes based on our last call, we'll now have a default config path; presumably `data`, if there's no config path we'll default to the data path.

File: .env.example [hunk 1]
```diff
@@ -16,3 +16,5 @@ APP_API_SERVER_URL=http://localhost:3000;http://api-dev.manuscripts.io:3000;http
 APP_ENABLE_NON_CONNECT_AUTH=0
 APP_DATABASE_URL=postgresql://postgres:admin@postgres:5432/test?schema=public
 CONFIG_PATH=./config
+CONFIG_DEFAULT=./data
+SECTION_CATEGORIES=./section-categories
```
[the remark this answers is not in the document]
Comment (mbetamony) [reply .env.example]: Got it, done.

File: data/templates.json [hunk 2]
```diff
@@ -4,31 +4,27 @@
     "objectType": "MPManuscriptTemplate",
     "bundle": "MPBundle:www-zotero-org-styles-american-medical-association",
     "title": "Nature Genetics Journal Publication",
-    "createdAt": 1433095330.89114,
-    "updatedAt": 1433095363.40453
+    "sectionCategories": "default-section-categories.json"
```
Comment (mnatsheh) [inline data/templates.json:7]: - I think we should allow the value to be an array directly as well.
- If the value is a string, we should treat it as a path, and resolve that path relative to `DATA_PATH`.
- We should have the value be clearly a path, something like "@default-section-categories.json" or "./default-section-categories.json".

File: src/DomainServices/ProjectService.ts [hunk 3]
```diff
@@ -83,19 +83,21 @@ export class ProjectService {
...
+      throw new MissingTemplateError(templateID)
     }
 
     const jats = await this.convert(file.path)
 
     const now = Math.round(Date.now() / 1000)
-    const { node, journal } = parseJATSArticle(jats, templateID)
+    const { node, journal } = parseJATSArticle(
+      jats,
+      JSON.parse(template).sectionCategories,
```
Comment (mnatsheh) [inline src/DomainServices/ProjectService.ts:98]: Do we actually need to reparse this?

Comment (mbetamony) [reply src/DomainServices/ProjectService.ts:98]: We save the data and serve it as a JSON string. We can change that if you like.

Comment (mnatsheh) [reply src/DomainServices/ProjectService.ts:98]: Let's keep it as is for now.

**Review by mnatsheh — APPROVED** [review]

**Pull request merged**